### PR TITLE
To Do list pull request

### PR DESCRIPTION
**Pull request by Grifo89**
### This pull request has the following features:
- Dinamically creates todos.
- Create separate projects.
- Delete, Edit and Change the status of the todo.
- View all todos from each project at once.
- Change color depends on priority.
- Use localStorage.

### CODE REVIEW

File: src/logic.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import objects from './objects';
+import dom from './domStuff';
+
+const projectExists = (title) => {
+  const data = localStorage;
+  let exists = true;
+  if (!data[title]) {
+    exists = false;
+  }
+  return exists;
```
Comment (mcrd25) [inline src/logic.js]: [OPTIONAL]
Code be shortened to `return !data[title] ? false : true;`

Comment (mcrd25) [reply src/logic.js]: Try and see other instances where code like this code be shortened

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+import './seed';
+
+const domElements = dom.domComponents();
+
+const alertExists = () => {
+  let exists = false;
+  const alert = document.querySelector('.red-alert p');
+  if (alert) {
+    exists = true;
+  }
```
Comment (mcrd25) [inline src/index.js]: [OPTIONAL]
This can be shortened to return `return alert ? true: false;`

**Comment by Grifo89** [comment]
Hi @mcrd25, regarding the stickler error, it was due to a Github's server error.

File: src/logic.js (hunk 1 above)
Comment (ViriCruz) [reply src/logic.js]: Thank you for the advice! 👏

**Pull request merged**